packages/salesforcedx-sobjects-faux-generator/src/data/minSObjects.json

```diff
@@ -0,0 +1 @@
tModifiedBy"},{"type":"List<WorkAccessShare>","name":"UserOrGroup"},{"type":"List<WorkBadge>","name":"CreatedBy"},{"type":"List<WorkBadge>","name":"Giver"},{"type":"List<WorkBadge>","name":"LastModifiedBy"},{"type":"List<WorkBadgeDefinition>","name":"CreatedBy"},{"type":"List<WorkBadgeDefinition>","name":"LastModifiedBy"},{"type":"List<WorkBadgeDefinition>","name":"Owner"},{"type":"List<WorkBadgeDefinitionFeed>","name":"CreatedBy"},{"type":"List<WorkBadgeDefinitionFeed>","name":"InsertedBy"},{"type":"List<WorkBadgeDefinitionHistory>","name":"CreatedBy"},{"type":"List<WorkBadgeDefinitionShare>","name":"LastModifiedBy"},{"type":"List<WorkBadgeDefinitionShare>","name":"UserOrGroup"},{"type":"List<WorkOrder>","name":"CreatedBy"},{"type":"List<WorkOrder>","name":"LastModifiedBy"},{"type":"List<WorkOrder>","name":"Owner"},{"type":"List<WorkOrderFeed>","name":"CreatedBy"},{"type":"List<WorkOrderFeed>","name":"InsertedBy"},{"type":"List<WorkOrderHistory>","name":"CreatedBy"},{"type":"List<WorkOrderLineItem>","name":"CreatedBy"},{"type":"List<WorkOrderLineItem>","name":"LastModifiedBy"},{"type":"List<WorkOrderLineItemFeed>","name":"CreatedBy"},{"type":"List<WorkOrderLineItemFeed>","name":"InsertedBy"},{"type":"List<WorkOrderLineItemHistory>","name":"CreatedBy"},{"type":"List<WorkOrderLineItemStatus>","name":"CreatedBy"},{"type":"List<WorkOrderLineItemStatus>","name":"LastModifiedBy"},{"type":"List<WorkOrderShare>","name":"LastModifiedBy"},{"type":"List<WorkOrderShare>","name":"UserOrGroup"},{"type":"List<WorkOrderStatus>","name":"CreatedBy"},{"type":"List<WorkOrderStatus>","name":"LastModifiedBy"},{"type":"List<WorkThanks>","name":"CreatedBy"},{"type":"List<WorkThanks>","name":"LastModifiedBy"},{"type":"List<WorkThanks>","name":"Owner"},{"type":"List<WorkThanksShare>","name":"LastModifiedBy"},{"type":"List<WorkThanksShare>","name":"UserOrGroup"},{"type":"List<WorkType>","name":"CreatedBy"},{"type":"List<WorkType>","name":"LastModifiedBy"},{"type":"List<WorkType>","name":"Owner"},{"type":"List<WorkTypeFeed>","name":"CreatedBy"},{"type":"List<WorkTypeFeed>","name":"InsertedBy"},{"type":"List<WorkTypeGroup>","name":"CreatedBy"},{"type":"List<WorkTypeGroup>","name":"LastModifiedBy"},{"type":"List<WorkTypeGroup>","name":"Owner"},{"type":"List<WorkTypeGroupFeed>","name":"CreatedBy"},{"type":"List<WorkTypeGroupFeed>","name":"InsertedBy"},{"type":"List<WorkTypeGroupHistory>","name":"CreatedBy"},{"type":"List<WorkTypeGroupMember>","name":"CreatedBy"},{"type":"List<WorkTypeGroupMember>","name":"LastModifiedBy"},{"type":"List<WorkTypeGroupMemberFeed>","name":"CreatedBy"},{"type":"List<WorkTypeGroupMemberFeed>","name":"InsertedBy"},{"type":"List<WorkTypeGroupMemberHistory>","name":"CreatedBy"},{"type":"List<WorkTypeGroupShare>","name":"LastModifiedBy"},{"type":"List<WorkTypeGroupShare>","name":"UserOrGroup"},{"type":"List<WorkTypeHistory>","name":"CreatedBy"},{"type":"List<WorkTypeShare>","name":"LastModifiedBy"},{"type":"List<WorkTypeShare>","name":"UserOrGroup"}]}]
```

Comment:
There are some object level data that we need to include here. I'm sure about the `custom` attribute we use here (link below) but there might be more . https://github.com/forcedotcom/salesforcedx-vscode/blob/053995436246ab97c91c98da9ccdb8411e65f6b9/packages/salesforcedx-sobjects-faux-generator/src/generator/fauxClassGenerator.ts#L210